playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -68,7 +65,7 @@
     private PKEvent.Listener eventListener;
     private PlayerEngine.EventListener eventTrigger = initEventListener();
     private PlayerEngine.StateChangedListener stateChangedTrigger = initStateChangeListener();
-
+    private final String nullPlayerMsgFormat = "Attempt to invoke '%s' on null instance of the player engine";
```

Comment:
I think it's better to keep this string inside the assert method because it's confusing here.